docs/cip/CIP-X_System_Info.md

```diff
@@ -0,0 +1,38 @@
+# CIP-X System Info
+
+## Status
+
+Current Status: `Under Discussion`
+
+## Motivation
+
+Currently, a lot of the support discussions in Discord revolve around gathering information about the user's operating
+environment. This information is crucial for debugging and troubleshooting issues. We want to make it easier for users
+to provide this information.
+
+## Public Interfaces
+
+This proposal introduces a new `API` method `system_info` that will return a dictionary with system information based on
```

Comment:
naming nit - I prefer .env() as it feels a little cleaner to type

Reply:
Done. Renamed across the board to `env`.